lib/dl_connector_postgresql/dl_connector_postgresql/formula/definitions/functions_array.py

```diff
@@ -369,4 +407,29 @@ def _array_notcontains(array: ClauseElement, value: ClauseElement) -> ClauseElem
             )
         ]
     ),
+    # arr_index_of
+    base.FuncArrayIndexOfStr(
+        variants=[
+            V(
+                D.POSTGRESQL,
+                lambda array, value: _array_index_of(array, sa.cast(value, sa.Text)),
```

Comment:
implicit cast, same issue as in https://github.com/datalens-tech/datalens-backend/pull/1196#discussion_r2288445091